protocol-designer/src/components/organisms/AssignLiquidsModal/index.tsx

```diff
@@ -130,55 +165,66 @@ export function AssignLiquidsModal(
                 })}
               </StyledText>
             </Flex>
-            <Box
-              width="100%"
-              padding={`${SPACING.spacing32} ${SPACING.spacing48}`}
-              backgroundColor={COLORS.white}
-              borderRadius={BORDERS.borderRadius12}
-              display={DISPLAY_GRID}
-              gap={SPACING.spacing12}
+            <Flex
+              flexDirection={DIRECTION_ROW}
+              gap={SPACING.spacing24}
+              position={POSITION_RELATIVE}
             >
-              <Flex flexDirection={DIRECTION_COLUMN} gap={SPACING.spacing16}>
-                <Flex
-                  justifyContent={JUSTIFY_CENTER}
-                  width="100%"
-                  color={COLORS.grey60}
-                >
-                  <StyledText
-                    desktopStyle="headingSmallRegular"
-                    css={{ userSelect: 'none' }}
+              {showLiquidLayoutOverlay && (
+                <LiquidLayoutOverlayModalContainer
+                  showLiquidOverflowMenu={setShowLiquidLayoutOverlay}
+                />
+              )}
+              <Box
+                width="100%"
+                padding={`${SPACING.spacing32} ${SPACING.spacing48}`}
+                backgroundColor={COLORS.white}
+                borderRadius={BORDERS.borderRadius12}
+                display={DISPLAY_GRID}
+                gap={SPACING.spacing12}
+              >
+                <Flex flexDirection={DIRECTION_COLUMN} gap={SPACING.spacing16}>
+                  <Flex
+                    justifyContent={JUSTIFY_CENTER}
+                    width="100%"
+                    color={COLORS.grey60}
                   >
-                    {t('click_and_drag')}
-                  </StyledText>
+                    <StyledText
+                      desktopStyle="headingSmallRegular"
+                      css={{ userSelect: 'none' }}
```

Comment:
we should avoid adding lines that use `styled-components`

```suggestion
                      style={{ userSelect: 'none' }}
```